cmd/helm/dependency.go

```diff
@@ -59,8 +59,8 @@ The 'version' field should contain a semantic version or version range.
 
 The 'repository' URL should point to a Chart Repository. Helm expects that by
 appending '/index.yaml' to the URL, it should be able to retrieve the chart
-repository's index. Note: 'repository' cannot be a repository alias. It must be
-a URL.
+repository's index. Note: 'repository' can be an alias. The alias must start
+with 'alias:' or '@'.
```

Comment:
I noticed that this whole block of documentation uses quotation marks inconsistently: here, single quotes, elsewhere double quotes for words larger than a single character. That's fodder for a later cross-cutting documentation scrub.